e2e/e2e_test.go

```diff
@@ -54,21 +54,66 @@ func TestE2EBasic(t *testing.T) {
 	config.ContainerService.Properties.CertificateProfile.CaCertificate = decodeCert(config.ContainerService.Properties.CertificateProfile.CaCertificate)
 	config.ContainerService.Properties.CertificateProfile.APIServerCertificate = decodeCert(config.ContainerService.Properties.CertificateProfile.APIServerCertificate)
 	config.ContainerService.Properties.CertificateProfile.ClientPrivateKey = decodeCert(config.ContainerService.Properties.CertificateProfile.ClientPrivateKey)
-	
+
 	// customData
 	baker := agent.InitializeTemplateGenerator()
 	base64EncodedCustomData := baker.GetNodeBootstrappingPayload(config)
 	customDataBytes, _ := base64.StdEncoding.DecodeString(base64EncodedCustomData)
 	customData := string(customDataBytes)
-	err := ioutil.WriteFile("scenarios/" + scenario + "/" + scenario + "-cloud-init.txt", []byte(customData), 0644)
+	err := ioutil.WriteFile("scenarios/"+scenario+"/"+scenario+"-cloud-init.txt", []byte(customData), 0644)
+	if err != nil {
+		fmt.Println("couldnt write to file", err)
+	}
+
+	// cseCmd
+	cseCommand := baker.GetNodeBootstrappingCmd(config)
+	err = ioutil.WriteFile("scenarios/"+scenario+"/"+scenario+"-cseCmd", []byte(cseCommand), 0644)
+	if err != nil {
+		fmt.Println("couldnt write to file", err)
+	}
+}
+
+func TestE2EWindows(t *testing.T) {
+	if os.Getenv("E2E_TEST") != "" {
+		t.Skip("This test needs e2e-script.sh to run first")
+	}
+
+	entry := "Generating CustomData and cseCmd"
+	fmt.Println(entry)
+
+	var scenario string = os.Getenv("SCENARIO_NAME")
+	fmt.Printf("Running for %s", scenario)
+
+	createFile("../e2e/scenarios/" + scenario + "/" + scenario + "-cloud-init.txt")
+	createFile("../e2e/scenarios/" + scenario + "/" + scenario + "-cseCmd")
+
+	nbc, _ := ioutil.ReadFile("scenarios/" + scenario + "/" + "nbc-" + scenario + ".json")
```

Comment:
what does nbc mean?

Reply:
It's an abbreviation from linux code and I guess it means "node bootstrapping config".